h2o-docs/src/product/data-science/gbm.rst

```diff
@@ -358,6 +358,21 @@ Metrics
 
 Usage is illustrated in the Examples section.
 
+GBM Friedman and Popescu's H statistics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can calculates the Friedman and Popescu's H statistics to test for the presence of an interaction between specified variables. 
+
+H varies from 0 to 1. It will have a value of 0 if the model exhibits no interaction between specified variables and a correspondingly larger value for a stronger interaction effect between them. NaN is returned if a computation is spoiled by weak main effects and rounding errors.
+
+This statistic can only be calculated for numerical variables. Missing values are supported. 
+
+See Jerome H. Friedman and Bogdan E. Popescu, 2008, "Predictive learning via rule ensembles", *Ann. Appl. Stat.* **2**:916-954, http://projecteuclid.org/download/pdfview_1/euclid.aoas/1223908046, s. 8.1.
```

Comment:
Can this be added to the reference section on the algo page or does it need to be in this section??

Reply:
Done. Thanks!